lua/neo-tree/sources/filesystem/commands.lua

```diff
@@ -68,10 +68,7 @@ M.delete_visual = function(state, selected_nodes)
 end
 
 M.expand_all_nodes = function(state)
-  local toggle_dir_no_redraw = function(_state, node)
-    fs.toggle_directory(_state, node, nil, true, true)
-  end
-  cc.expand_all_nodes(state, toggle_dir_no_redraw)
+  cc.expand_all_nodes(state, state.tree:get_node(state.path))
```

Comment:
Thanks to such interface expanding all nodes recursively under the cursor will be as simple as `expand_all_nodes(state, state.tree:get_node())`.